docs/maintain/maintain-guides-how-to-stop-validating.md

```diff
@@ -28,17 +28,25 @@ To chill your validator or nominator, call the `staking.chill()` extrinsic. See
 
 ## Purge validator session keys
 
-Purging the validator's session keys removes the key reference. This can be done
-through the `session.purgeKeys()` extrinsic either from the controller or from the stash account.
+Purging the validator's session keys removes the key reference. This can be done through the
+`session.purgeKeys()` extrinsic. The key reference exists on the account that originally called the
+`session.set_keys()` extrinsic, which could be the stash or the controller (at the time the keys
+were set).
 
-:::caution 
+:::caution Purge keys using the same account that set the keys
 
-The key reference exists on the account that called the `session.set_keys()` extrinsic when the keys were set. That could be the stash or the controller (at the time). **It is the same account that needs to call the `session.purge_keys()` extrinsic** in order for the correct reference to be removed. Calling the `session.purge_keys()` from the wrong account, although it may succeed, will result in a reference on the other account that **cannot** be removed, and as a result that account won't be possible to be reaped.
+**It is the same account that needs to call the `session.purge_keys()` extrinsic** in order for the
```

Comment:
I'm not sure it's clear who "the same account" refers to. 

Probably needs to be reworded to:
"**Make sure to call the `session.purge_keys()` extrinsic from the same account that set the keys in the first place** in order for the..."